source/common.c

```diff
@@ -16,6 +16,11 @@
 #include <aws/common/common.h>
 #include <stdlib.h>
 
+#ifdef _MSC_VER 
+#pragma warning( push )
+#pragma warning( disable : 4100)
```

Comment:
What warning are we suppressing?

Reply:
unused function argument warning for windows, since visual c doesn't technically have a "c" compiler, it carries this one over from C++.

Reply:
I'll add a comment.